<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SampleShapePropertyWidget</class>
 <widget class="QWidget" name="SampleShapePropertyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2">
     <item>
      <widget class="QPushButton" name="pushButton_LoadMovie">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Load a info file containing Video&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton:hover{
        background: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 :   1, stop :   0.0 #ffd9aa,
                stop :   0.5 #ffbb6e, stop :   0.55 #feae42, stop :   1.0 #fedb74);
}

QPushButton {
        border: 1px solid #6593cf;
        border-radius: 2px;
        padding: 5px 15px 2px 5px;
        background: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 :   1, stop :   0.0 #f5f9ff,
                stop :   0.5 #c7dfff, stop :   0.55 #afd2ff, stop :   1.0 #c0dbff);
        color: #006aff;
        font: bold large &quot;Arial&quot;;
        height: 30px;
}




QPushButton:pressed {
        background: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 :   1, stop :   0.0 #c0dbff,
        stop :   0.5 #cfd26f, stop :   0.55 #c7df6f, stop :   1.0 #f5f9ff);
        padding-top: 2px;
        padding-left: 3px;

}


QPushButton:on {
        background: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 :   1, stop :   0.0 #5AA72D,
        stop :   0.5 #B3E296, stop :   0.55 #B3E296, stop :   1.0 #f5f9ff);
        padding-top: 2px;
        padding-left: 3px;
}

QPushButton:disabled {
        background: transparent #e5e9ee;
        padding-top: 2px;        
        padding-left: 3px;
        color: black;
}
</string>
       </property>
       <property name="text">
        <string>Load Crystal Movie</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_MovieFilename">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 1px solid blue;
    border-radius: 10px;
    padding: 0 8px;
    background: white;
    selection-background-color: darkgray;
}</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Volume:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_Volume">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 1px solid blue;
    border-radius: 10px;
    padding: 0 8px;
    background: white;
    selection-background-color: darkgray;
}</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Faces:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_Faces">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 1px solid blue;
    border-radius: 10px;
    padding: 0 8px;
    background: white;
    selection-background-color: darkgray;
}</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Edges:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_Edges">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 1px solid blue;
    border-radius: 10px;
    padding: 0 8px;
    background: white;
    selection-background-color: darkgray;
}</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Vertices:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_Vertices">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 1px solid blue;
    border-radius: 10px;
    padding: 0 8px;
    background: white;
    selection-background-color: darkgray;
}</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_MovieFilename</tabstop>
  <tabstop>lineEdit_Volume</tabstop>
  <tabstop>lineEdit_Faces</tabstop>
  <tabstop>lineEdit_Edges</tabstop>
  <tabstop>lineEdit_Vertices</tabstop>
  <tabstop>pushButton_LoadMovie</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
